test/real_cases_dichotomious_test.rb

```diff
@@ -18,7 +18,7 @@
 require './test/test_helper'
 
 class RealCasesTest < Minitest::Test
-  if !ENV['SKIP_REAL_DICHO'] && !ENV['SKIP_DICHO'] && !ENV['TRAVIS']
```

Comment:
If we need to skip this test on CI we can replace with `ENV['CI']`

Reply:
*RealCasesTest* must be launched in pipeline ? If not we must add `&& !ENV['CI']`